stack_integration_test.go

```diff
@@ -0,0 +1,172 @@
+package tfe
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestStackCreateAndList(t *testing.T) {
+	skipUnlessBeta(t)
+
+	client := testClient(t)
+	ctx := context.Background()
+
+	orgTest, orgTestCleanup := createOrganization(t, client)
+	t.Cleanup(orgTestCleanup)
+
+	project2, err := client.Projects.Create(ctx, orgTest.Name, ProjectCreateOptions{
+		Name: "test-project-2",
+	})
+	require.NoError(t, err)
+
+	oauthClient, cleanup := createOAuthClient(t, client, orgTest, nil)
+	t.Cleanup(cleanup)
+
+	stack1, err := client.Stacks.Create(ctx, StackCreateOptions{
+		Name: "aa-test-stack",
+		VCSRepo: &StackVCSRepo{
+			Identifier:   "brandonc/pet-nulls-stack",
+			OAuthTokenID: oauthClient.OAuthTokens[0].ID,
+		},
+		Project: &Project{
+			ID: orgTest.DefaultProject.ID,
+		},
+	})
```

Comment:
Let's refactor this into a test helper `createStack()`

Reply:
Eh I think I'll wait on this one